cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -226,3 +227,49 @@ message SegYSeismicResponse {
 message SearchFilesRequest {
     SearchSpec spec = 1;
 }
+
+message SearchJobStatusRequest {
+    message Filter {
+        message Id {
+            oneof id {
+                string job_id = 1;
+                string file_id = 2;
+                string file_name = 3;
+            }
+        }
+
+        oneof filter {
+            Id id = 1;
```

Comment:
Filtering by job_id will limit results to 1, whereas file_id or name can potentially return multiple jobs